modules/37_industry/subsectors/equations.gms

```diff
@@ -211,4 +215,106 @@ q37_costCESmarkup(t,regi,in)$(ppfen_industry_dyn37(in))..
+
+* feedstocks flow has to be lower than total energy flow into industry
+q37_feedstocksLimit(ttot,regi,entySE,entyFE,emiMkt)$(
+                                             ttot.val ge cm_startyear       
+                                         AND sefe(entySE,entyFE)
+                                         AND sector2emiMkt("indst",emiMkt) 
+                                         AND entyFe2Sector(entyFe,"indst") 
+                                         AND entyFeCC37(entyFe)            ) ..
+  vm_demFESector(ttot,regi,entySE,entyFE,"indst",emiMkt)
+  =g=
+  vm_demFENonEnergySector(ttot,regi,entySE,entyFE,"indst",emiMkt)  
+;
+
+*** calculate carbon contained in chemical feedstocks
+q37_FeedstocksCarbon(ttot,regi,sefe(entySE,entyFE),emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt) ) ..
+  vm_FeedstocksCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  =e=
+    vm_demFENonEnergySector(ttot,regi,entySE,entyFE,"indst",emiMkt)
+  * p37_FeedstockCarbonContent(ttot,regi,entyFE)
+;
+
+*** calculate carbon contained in plastics [GtC]
+q37_plasticsCarbon(ttot,regi,sefe(entySE,entyFE),emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt) ) ..
+  vm_plasticsCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  =e=
+    vm_FeedstocksCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  * s37_plasticsShare
+;
+
+*** calculate emissions from plastics incineration
+q37_incinerationEmi(ttot,regi,sefe(entySE,entyFE),emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt) ) ..
+  vm_incinerationEmi(ttot,regi,entySE,entyFE,emiMkt)
+  =e=
+    vm_plasticsCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  * pm_incinerationRate(ttot,regi)
+;
+
+*** calculate carbon contained in non-incinerated plastics
+*** this is used in emissions accounting to subtract the carbon that gets
+*** sequestered in plastic products
+q37_nonIncineratedPlastics(ttot,regi,sefe(entySE,entyFE),emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt) ) ..
+  vm_nonIncineratedPlastics(ttot,regi,entySE,entyFE,emiMkt)
+  =e=
+    vm_plasticsCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  * (1 - pm_incinerationRate(ttot,regi))
+  ;
+
+*** calculate flow of carbon contained in chemical feedstock with unknown fate
+*** it is assumed that this carbon is re-emitted in the same timestep 
+q37_feedstockEmiUnknownFate(ttot,regi,sefe(entySE,entyFE),emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt) ) ..
+  vm_feedstockEmiUnknownFate(ttot,regi,entySE,entyFE,emiMkt)
+  =e=
+    vm_FeedstocksCarbon(ttot,regi,entySE,entyFE,emiMkt)
+  * (1 - s37_plasticsShare)
+;
+
+*** in baseline runs, all industrial feedstocks should come from fossil energy
+*** carriers, no biofuels or synfuels
+q37_FossilFeedstock_Base(t,regi,entyFE,emiMkt)$(
+                         entyFE2sector2emiMkt_NonEn(entyFE,"indst",emiMkt)
+                     AND cm_emiscen eq 1                                   ) ..
```

Comment:
Not sure if that is (still) sufficient, ince we more or less replaced baseline by NPi.
Should this be the case for NPi runs, too?
See https://github.com/remindmodel/remind/pull/1342.